com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateOrUpdateAppStep.java

```diff
@@ -171,6 +173,10 @@ public void handleApplicationAttributes() {
             }
             client.createApplication(app.getName(), app.getStaging(), diskQuota, memory, uris, Collections.emptyList(),
                                      app.getDockerInfo());
+            CloudApplication application = client.getApplication(app.getName());

```

Comment:
Why is this duplicated?
First here in handleAppAtributes, then again in handleAppMetadata?